netfox/Core/NFXNetService.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  NFXNetService.swift
+//  netfox_mac
+//
+//  Created by Alexandru Tudose on 01/11/2017.
+//  Copyright © 2017 kasketis. All rights reserved.
+//
+
+import Foundation
+
+#if os(OSX)
+class NFXNetService: NSObject {
```

Comment:
Maybe call this class `NFXNetServiceMonitor`, or something that indicates it monitors the services?